mono/tests/bug-gh-9507.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+
+public struct TestIsInst<T>
+{
+        public T[] inArray1;
+
+        public TestIsInst(int i) {
```

Comment:
Fix indentation to respect rest of codebase. It might be a test but it's always kinder on the eye to have code formatting consistency.